shell/platform/embedder/embedder.h

```diff
@@ -1368,6 +1368,15 @@ typedef struct {
   /// matches what the platform would natively resolve to as possible.
   FlutterComputePlatformResolvedLocaleCallback
       compute_platform_resolved_locale_callback;
+
+  /// The command line argument count for arguments passed through to the Dart
+  /// entrypoint.
+  int dart_entrypoint_argc;
+
+  /// The command line arguments passed through to the Dart entrypoint. The
+  /// strings must be `NULL` terminated.
+  const char* const* dart_entrypoint_argv;
```

Comment:
Please document that the null-terminated string will be copied out and can be collected after the call to initialize the engine with the `FlutterProjectArgs`.